File: frontend/lib/requests/create_expense_request.dart
class ExpenseCreateRequest {
  final double price;
  final String currency;
  final String description;
  final int creatorId;
  final int journeyId;
  final List<int> participants;

  ExpenseCreateRequest(
      {required this.price,
      required this.currency,
      required this.description,
      required this.creatorId,
      required this.journeyId,
      required this.participants});

  Map<String, dynamic> toJson() {
    return {
      'price': price,
      'creatorId': creatorId,
      'journeyId': journeyId,
      'currency': currency,
      'description': description,
      'participants': participants
    };
  }
}

File: Backend/Iteramico/DomainService/Requests/Expense/CreateExpenseRequest.cs
namespace DomainService.Requests.Expense
{
#nullable disable
    public class CreateExpenseRequest
    {
        public decimal Price { get; set; }
        public string Currency { get; set; }
        public string Description { get; set; }
        public int CreatorId {  get; set; }
        public int JourneyId { get; set; }
        public List<int> Participants { get; set; }
    }
}